embeddable-say-editor/app/components/simple-editor.js

```diff
@@ -469,8 +469,12 @@ export default class SimpleEditorComponent extends Component {
 
   setupLpdcPlugin(setup) {
     const { config, userConfig } = setup;
-
-    config.lpdc = userConfig.lpdc;
+    const endpoint = userConfig.lpdc.endpoint
+      ? userConfig.lpdc.endpoint + '/doc/instantie'
+      : userConfig.lpdc.lpdcEndpoint;
+    config.lpdc = {
+      endpoint,
```

Comment:
this breaks the config, cause you're not passing all the options...
there is also decisionUri, as you can see from the docs
but better to spread the rest of the config